### ISSUE TITLE
The accuracy is severely reduced after converting to onnx

### ISSUE THREAD

**Issue by JF-Lee** [onnx/onnx#3015]
I tried to use mobilenetv3 for color classification, but the accuracy was severely reduced after I converted to onnx.

acc before  onnx:
![image](https://user-images.githubusercontent.com/66176517/92995109-de999b00-f532-11ea-9002-af001f21a2cd.png)

acc after onnx:
![image](https://user-images.githubusercontent.com/66176517/92995121-f40ec500-f532-11ea-8ffe-a212ef2b5bf0.png)

code for color detecting:
    def detect(self, weight_path, pic_path, f):
        self.load_weights(weight_path=weight_path)

        img0 = Image.open(pic_path).convert('RGB')
        img = np.uint8(img0)
        im = cv2.cvtColor(np.uint8(img), cv2.COLOR_RGB2Lab)
        img = Image.fromarray(im)

        transform = transforms.Compose([
            transforms.Resize((224, 224)),
            transforms.ToTensor(),
        ])
        img_tensor = transform(img).unsqueeze(0)
        if torch.cuda.is_available():
            img_tensor = img_tensor.cuda()
        net_output = self.net(img_tensor)

        _, predicted = torch.max(net_output.data, 1)

        result = predicted[0].item()

        color = ['black', 'blue', 'green', 'red', 'white', 'yellow']

        return color[result]

Hope to get your answer, thank you!!!

**Comment by jcwchen** [comment]
Hi @JF-Lee,
Did you use onnxruntime to inference the onnx model? Please provide the onnx model and the inference script to reproduce the problem if possible. Thank you.

**Comment by JF-Lee** [comment]
I used ort for inference，and the project file is attached to the baidu link.
链接：https://pan.baidu.com/s/1TElJboHp8wf-jTAONsLxjg 
提取码：88yl



You need to unzip the color_val.zip and run the inference.py to get the acc of the original pytorch pretrained model.
Thanks for your help!!!






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "onnx/onnx"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月13日(星期天) 凌晨5:56
收件人:&nbsp;"onnx/onnx"<onnx@noreply.github.com&gt;;
抄送:&nbsp;"走走停停ゝ"<445099619@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [onnx/onnx] The accuracy is severely reduced after converting to onnx (#3015)





 
Hi @JF-Lee,
 Did you use onnxruntime to inference the onnx model? Please provide the onnx model and the inference script to reproduce the problem if possible. Thank you.

**Comment by jcwchen** [comment]
Since you didn't provide the test data in the directory, I tested some images on my own. It seems that the ort.inference will inference the same color with any kinds of images... It might be an issue in `torch.onnx._export` or onnxruntime. As far as I know, onnxruntime does support mobilenetv3. Therefore, I would suggest you open an issue under pytorch repo about this to get the best help. Thanks.

**Comment by JF-Lee** [comment]
Thank you very much for your help！As you have tested，the ort.inference will inference the same color with any kinds of images, I will open an issue under ptorch repo about this! Thanks again!!!





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "onnx/onnx"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月15日(星期二) 凌晨3:30
收件人:&nbsp;"onnx/onnx"<onnx@noreply.github.com&gt;;
抄送:&nbsp;"走走停停ゝ"<445099619@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [onnx/onnx] The accuracy is severely reduced after converting to onnx (#3015)





 
Since you didn't provide the test data in the directory, I tested some images on my own. It seems that the ort.inference will inference the same color with any kinds of images... It might be an issue in torch.onnx._export or onnxruntime. As far as I know, onnxruntime does support mobilenetv3. Therefore, I would suggest you open an issue under pytorch repo about this to get the best help. Thanks.

**Comment by askhade** [comment]
Closing this issue here. Please reopen it if you have more concerns.
